### ISSUE TITLE
[Feature] Don't check services when -X passed on debian family

### ISSUE THREAD

**Issue by gtmanfred** [saltstack/salt-bootstrap#1140]
### Description of Issue/Question

I know that ubuntu/debian always start the services when they get installed.  But if `-X` is passed, we don't need to check the services.

Thanks,
Daniel

**Comment by rallytime** [comment]
Sure. Should be easy to fix.

**Comment by vutny** [comment]
@gtmanfred @rallytime I've verified both Debian and Ubuntu installation against latest `develop`, and the `-X` option really does what it is supposed to do: skips checking if services are **running** after installation.

If you meant that we should skip checking if services are **enabled to start** after installation, that's different case and it supposed to be triggered by `-d` option. And I've found that such check is also being skipped as expected, but across all the distros Salt services are going to be enabled in any case.

I've fixed it in my own fork of the bootstrap script, and I think I could try to merge it back to the upstream.
